examples/monaco-graphql-nextjs/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "next-monaco-example",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@graphiql/toolkit": "^0.8.0",
+    "@next/font": "13.1.6",
+    "@types/node": "18.13.0",
+    "@types/react": "18.0.28",
+    "@types/react-dom": "18.0.11",
+    "@zeit/next-css": "^1.0.1",
```

Comment:
this package is deprecated 
<img width="1234" alt="image" src="https://user-images.githubusercontent.com/7361780/221322964-84749edf-5241-43ac-b69d-407f2b52beee.png">

Reply:
oops! that was leftover from an experiment, it isn't being used